lms/templates/dashboard/_dashboard_certificate_information.html

```diff
@@ -30,7 +30,7 @@
 <div class="message message-status ${status_css_class} is-shown">
 % if cert_status['status'] == 'processing':
   <p class="message-copy">${_("Final course details are being wrapped up at this time. Your final standing will be available shortly.")}</p>
-% elif cert_status['status'] in ('generating', 'ready', 'notpassing', 'restricted', 'auditing'):
+% elif cert_status['status'] in ('generating', 'ready', 'notpassing', 'restricted', 'auditing', 'unverified'):
```

Comment:
Would be nice to tie-in CertificateStatuses here, or better yet push this conditional into the view and provide a context parameter instead that you can display if/when populated.